go/teams/loader.go

```diff
@@ -309,6 +309,15 @@ func (l *TeamLoader) load1(ctx context.Context, me keybase1.UserVersion, lArg ke
 		}
 	}
 
+	if l.G().Env.GetFeatureFlags().HasFeature(libkb.FeatureBoxAuditor) {
+		newMctx, shouldReload := VerifyBoxAudit(libkb.NewMetaContext(ctx, l.G()), teamID)
```

Comment:
`VerifyBoxAudit` will return `shouldReload` after reauditing, is this necessary if auditing did not trigger rotating? Or am I missing something?

Reply:
Not necessary. AssertUnjailedOrReaudit will return `didReaudit=true` if it rotated, which happens iff it is reaudited.